plugins/schedule/base/lib/model/om/BaseScheduleEventResource.php

```diff
@@ -1192,8 +1192,16 @@ public function buildPkeyCriteria()
 			if ($this->isColumnModified(ScheduleEventResourcePeer::CUSTOM_DATA))
 			{
 				if (!is_null($this->custom_data_md5))
+				{
 					$criteria->add(ScheduleEventResourcePeer::CUSTOM_DATA, "MD5(cast(" . ScheduleEventResourcePeer::CUSTOM_DATA . " as char character set latin1)) = '$this->custom_data_md5'", Criteria::CUSTOM);
 					//casting to latin char set to avoid mysql and php md5 difference
+					if (kDataCenterMgr::isMultiDc()) // if multi DC configuration don't check costume data on other DC
```

Comment:
same as previous file

Reply:
@inbal-ben-david this is something that was added when running the propel-gen automatic ORM generation